test/apify/enqueue_links/enqueue_links.test.ts

```diff
@@ -223,25 +242,16 @@ describe('enqueueLinks()', () => {
                 /(http|https):\/\/cool\.com\//,
             ];
 
-            await browserCrawlerEnqueueLinks({
-                options: { selector: '.click', pseudoUrls },
-                page,
-                requestQueue,
-            });
-
-            expect(enqueued).toHaveLength(3);
-
-            expect(enqueued[0].url).toBe('https://example.com/a/b/first');
-            expect(enqueued[0].method).toBe('GET');
-            expect(enqueued[0].userData).toEqual({});
-
-            expect(enqueued[1].url).toBe('https://example.com/a/b/third');
-            expect(enqueued[1].method).toBe('GET');
-            expect(enqueued[1].userData).toEqual({});
-
-            expect(enqueued[2].url).toBe('http://cool.com/');
-            expect(enqueued[2].method).toBe('GET');
-            expect(enqueued[2].userData).toEqual({});
+            try {
+                await browserCrawlerEnqueueLinks({
+                    // @ts-expect-error Type 'RegExp[]' is not assignable to type 'PseudoUrlInput[]'
+                    options: { selector: '.click', pseudoUrls },
+                    page,
+                    requestQueue,
+                });
+            } catch (err) {
+                expect((err as Error).message).toMatch(/Expected values to be of type `string` but received type `RegExp`/);
+            }
```

Comment:
for testing error handling, you should prefer `expect(() => ....).toThrow()` as that way you know things will actually throw or the test will fail. the way you have it now would silently pass if the function would not throw.

Reply:
makes sense, will update, thx